src/loader/mod.rs

```diff
@@ -55,8 +50,8 @@ pub enum Error {
     CommandLineCopy,
     /// Command line overflowed guest memory.
     CommandLineOverflow,
-    /// Invalid ELF magic number
-    InvalidElfMagicNumber,
+    /// Invalid magic number
+    InvalidMagicNumber,
```

Comment:
I renamed it to from `InvalidElfMagicNumber` to `InvalidMagicNumber`, a more generic one. Because an error message is needed when parsing ARM kernel, but defining a new `InvalidPeMagicNumber` looks too verbose.